ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.5.yaml

```diff
@@ -56,3 +56,8 @@ tests:
   openshift_installer:
     cluster_profile: gcp
     upgrade: true
+- as: e2e-ovn-steps
```

Comment:
@JacobTanenbaum should this go into ./ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml (eg master) instead?

Reply:
done